plugin.php

```diff
@@ -167,21 +167,26 @@
 }
 
 // Return title custom by custom field onOffice
-function getRestrictLength( $valueNumber, $title ) {
-	if ( empty( $valueNumber ) ) {
+function getRestrictLength( $limitEllipsis, $title ,$ellipsis = '...' ) {
```

Comment:
Please use the proper ellipsis character, `…`, instead of three periods.

Reply:
For 2. you can manually clear the cache which will fix that.